src/Butler/Auth.hs

```diff
@@ -7,20 +7,24 @@ import Butler.Frame
 import Butler.Prelude
 import Lucid.XStatic
 
-publicDisplayApp :: DisplayApplication
-publicDisplayApp = DisplayApplication auth
+newtype PageTitle = PageTitle Text
+    deriving newtype (IsString, ToHtml)
+
+newtype PageDesc = PageDesc Text
+    deriving newtype (IsString)
+
+publicDisplayApp :: PageTitle -> Maybe PageDesc -> DisplayApplication
+publicDisplayApp appTitle appDescM = DisplayApplication auth
   where
-    auth xfiles sessions = guestAuthApp sessions $ htmlMain xfiles "Standalone GUI"
+    auth xfiles sessions = guestAuthApp sessions $ htmlMain xfiles appTitle appDescM
 
-htmlMain :: [XStaticFile] -> Text -> Html () -> Html ()
-htmlMain xfiles title body = do
+htmlMain :: [XStaticFile] -> PageTitle -> Maybe PageDesc -> Html () -> Html ()
+htmlMain xfiles title descM body = do
     doctypehtml_ do
         head_ do
             title_ (toHtml title)
-            meta_ [charset_ "utf-8"]
+            maybe mempty (\(PageDesc desc) -> meta_ [name_ "description", content_ desc]) descM
```

Comment:
Did you remove the charset option on purpose?
For such action that are empty on nothing, you can use this idiom `forM_ mValue \value ->`, e.g.

```suggestion
            forM_ mDesc \desk -> meta_ [name_ "description", content_ desc]
```

Reply:
Oh no that's a mistake. Ok yes better.